packages/help-center/src/components/help-center-contact-form.tsx

```diff
@@ -268,23 +268,18 @@ export const HelpCenterContactForm = () => {
 				return;
 			}
 
-			const searchResult = {
-				...result,
-				title: preventWidows( decodeEntities( result.title ) ),
-				query: debouncedMessage,
-			};
 			const params = new URLSearchParams( {
 				link: result.link,
-				postId: result.post_id,
+				postId: result.post_id.toString(),
 				query: debouncedMessage || '',
 				title: preventWidows( decodeEntities( result.title ) ),
 			} );
 
 			if ( result.blog_id ) {
-				params.set( 'blogId', result.blog_id );
+				params.set( 'blogId', result.blog_id.toString() );
 			}
 
-			navigate( `/post/?${ params }`, searchResult );
+			navigate( `/post/?${ params }` );
```

Comment:
As far as I can tell, search result doesn't do anything. I'm double checking with the original author to be sure, though.